test/functional-tests/profile.spec.js

```diff
@@ -459,94 +418,94 @@ describe('Profile and accounts workflow:', function () {
+  //     browser.click('.button-settings')
+  //     browser.waitForClickable('.edit-password')
+  //     browser.click('.edit-password')
+  //     browser.waitAndSetValue('#current-password', 'myshinynewpassword')
+  //     browser.click('#save-password')
+  //
+  //     assert.equal(browser.isVisible('.edit-password-modal'), true)
+  //     assert.equal(browser.getAttribute('#save-password', 'disabled'), 'true')
+  //   })
+  //
+  //   it.skip('should not allow the user to attempt to change their password if they do not enter a new password', function () {
+  //     createUserAndLogin(browser)
+  //     browser.url('http://localhost:3000/profile')
+  //     browser.waitAndClick('.button-settings')
+  //     browser.waitAndClick('.edit-password')
+  //     browser.waitAndSetValue('#current-password', 'myshinynewpassword')
+  //     browser.click('#save-password')
+  //
+  //     assert.equal(browser.isVisible('.edit-password-modal'), true)
+  //     assert.equal(browser.getAttribute('#save-password', 'disabled'), 'true')
+  //   })
+  //
+  //   it.skip('should not allow the user to attempt to change their password if they do not enter their current password or a new password', function () {
+  //     createUserAndLogin(browser)
+  //     browser.url('http://localhost:3000/profile')
+  //     browser.waitForClickable('.button-settings')
+  //     browser.click('.button-settings')
+  //     browser.waitForClickable('.edit-password')
+  //     browser.click('.edit-password')
+  //     browser.waitForClickable('#current-password')
+  //     assert.equal(browser.getAttribute('#save-password', 'disabled'), 'true')
+  //   })
+  //
+  //   it.skip('should allow the user to attempt to change their password if they enter the correct current password and a new password', function () {
+  //     createUserAndLogin(browser)
+  //     browser.url('http://localhost:3000/profile')
+  //     browser.waitForClickable('.button-settings')
+  //     browser.click('.button-settings')
+  //     browser.waitForClickable('.edit-password')
+  //     browser.click('.edit-password')
+  //     browser.waitAndSetValue('#current-password', 'password')
+  //     browser.waitAndSetValue('#new-password', 'foobar')
+  //     browser.waitForClickable('#save-password')
+  //     browser.click('#save-password')
+  //
+  //     browser.waitUntil(() => {
+  //       return !browser.isVisible('.edit-password-modal')
+  //     })
+  //     browser.pause(500)
+  //     browser.waitForClickable('#logout a')
+  //     browser.click('#logout a')
+  //     browser.waitForVisible('#confirmLogout')
+  //     browser.pause(500)
+  //     browser.waitForClickable('#logoutBtn')
+  //     browser.click('#logoutBtn')
+  //     browser.waitForClickable('#nav-profile')
+  //     assertUserIsNotLoggedIn(browser)
+  //
+  //     login(browser, 'foobar')
+  //
+  //     waitForUserIsLoggedIn(browser)
+  //     assertUserIsLoggedIn(browser)
+  //   })
+  // })
```

Comment:
Are these just temporarily commented out?

Reply:
we are not using so maybe we could remove it but it's a temporary reminder

Reply:
If we are _sure_ we are going to need this later I'd just add a `.skip`, otherwise delete